<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortieStock</class>
 <widget class="QDialog" name="SortieStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Produit:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtSortieProd"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Quantit√©:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtSortieQTE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date Sortie:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnReset4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[ Enregistrement d'une sortie de stock ]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
